### ISSUE TITLE
Trigger that fires too often

### ISSUE THREAD

**Issue by mversic** [hyperledger-iroha/iroha#5232]
it's possible to register a trigger that fires so often that it jams the network. This should be guarded against. When registering such triggers they should get rejected

**Comment by s8sato** [comment]
- Does a data trigger reacts to every data event in the event pool at the round? -- inspect
- Related to #4937 and #4968:
  One of the concepts of pooling events was preventing loop of trigger executions. If we want an event to be converted to an intrinsic transaction and inserted to the other ordered transactions without being pooled, it might come down to this issue

**Comment by s8sato** [comment]
> - Does a data trigger react to every data event in the event pool during the round?

Yes, #4937 should fix it to react only once to the event union per transaction
